docker/README.md

```diff
@@ -3,8 +3,3 @@
 This `docker/` directory serves the local docker development process by being a convenient place
 for [bind mounts](https://docs.docker.com/storage/bind-mounts/). Developers may structure data
 however they see fit in any subdirectories.
-
-By convention though, the main [docker-compose.yml](../docker-compose.yml) file will use the
-`docker/core/data/` directory as the bind mount for the `/data` volume and the
-`docker/core/cache/` directory as the bind mount for the `/cache` volume with respect to the
-`grpc-api` service.
```

Comment:
Do we need to indicate where `/data` is bind mounted?  We deleted the comments related to this.

Reply:
We could, but it seems awkward to describe the `./data` mount in the apparently unrelated `./docker/README.md` file. If we do write something, where should it live?

Reply:
Maybe this is what we want?

```
By convention though, the main[docker-compose.yml](../docker-compose.yml) file will use the `./data` directory as the bind mount for the `/data` volume and the `./docker/core/cache/` directory as the bind mount for the `/cache` volume with respect to the grpc-api service.
```

Reply:
Per chat, I've used the above text.